packages/pds/src/scripts/sequencer-recovery/index.ts

```diff
@@ -0,0 +1,14 @@
+import AppContext from '../../context'
+import { Recoverer } from './recoverer'
+
+export const sequencerRecovery = async (ctx: AppContext, args: string[]) => {
+  const cursor = args[0] ? parseInt(args[0]) : 0
+  const concurrency = args[1] ? parseInt(args[1]) : 10
```

Comment:
I know, super annoying 🙈 
```suggestion
  const cursor = args[0] ? parseInt(args[0], 10) : 0
  const concurrency = args[1] ? parseInt(args[1], 10) : 10
```

Even more annoying, might be nice to do a `!isNaN()` assertion on the inputs.

Reply:
nah i dig 👍